spring-boot/proxy/src/main/java/eu/coatrack/proxy/filters/post/ErrorLoggingFilter.java

```diff
@@ -50,6 +51,9 @@
 
     private static Logger log = LoggerFactory.getLogger(ErrorLoggingFilter.class);
 
+    @Value("${ygg.admin.server.url}")
```

Comment:
Should the parameter also be removed here, as you removed it on the GitService class?

Reply:
Done. Already removed.

Reply:
I checked and it is indeed removed, so this is done